packages/mds-db/geographies.ts

```diff
@@ -205,3 +236,16 @@ export async function updateGeographyMetadata(geography_metadata: GeographyMetad
     ...recorded_metadata
   }
 }
+
+export async function deleteGeographyMetadata(geography_id: UUID) {
```

Comment:
Should this `throw` if it doesn't find a corresponding `geography_id`?

Reply:
I don't think so. If there's a metadata without a geo, we should definitely be able to delete it.

Reply:
We should never have metadata without a corresponding geo though

Reply:
I agree but shit happens @avatarneil

Reply:
I agree but shit happens @avatarneil

Reply:
I agree but shit happens @avatarneil

Reply:
I agree but shit happens @avatarneil

Reply:
I agree but shit happens @avatarneil